packages/client/src/reactions/Reactions.ts

```diff
@@ -1,37 +1,47 @@
 import { GraphQLClient } from 'graphql-request';
 
-import { Credentials } from '../authentication';
+import { Authentication } from '../authentication';
 import { LensConfig } from '../consts/config';
 import { NotAuthenticatedError } from '../consts/errors';
 import { ReactionRequest } from '../graphql/types.generated';
 import { getSdk, Sdk } from './graphql/reactions.generated';
 
 export class Reactions {
+  private readonly authentication: Authentication | undefined;
   private readonly sdk: Sdk;
 
-  constructor(config: LensConfig, private readonly credentials?: Credentials) {
+  constructor(config: LensConfig, authentication?: Authentication) {
```

Comment:
`Authentication` is optional, we can have modules that contain both auth-required and public methods and inform user (currently by throwing an error) if misused.